x-pack/metricbeat/module/googlecloud/compute/_meta/docs.asciidoc

```diff
@@ -80,3 +80,17 @@ Here is a list of metrics collected by `compute` metricset:
 * `instance.network.sent_packets_count`: Count of packets sent over the network
 - `instance_name`: The name of the VM instance.
 - `loadBalanced`: Whether traffic was sent from an L3 loadbalanced IP address assigned to the VM. Traffic that is externally routed from the VM's standard internal or external IP address, such as L7 loadbalanced traffic, is not considered to be loadbalanced in this metric.
+
+[float]
+==== memory
+* `instance.memory.balloon.ram_size`: The total amount of memory in the VM. This metric is only available for VMs that belong to the e2 family.
+- `instance_name`: The name of the VM instance.
+
+* `instance.memory.balloon.ram_used`: Memory currently used in the VM. This metric is only available for VMs that belong to the e2 family.
+- `instance_name`: The name of the VM instance.
+
+* `instance.memory.balloon.swap_in_bytes_count`: The amount of memory read into the guest from its own swap space. This metric is only available for VMs that belong to the e2 family.
+- `instance_name`: The name of the VM instance.
+
+* `instance.memory.balloon.swap_out_bytes_count`: The amount of memory written from the guest to its own swap space. This metric is only available for VMs that belong to the e2 family.
+- `instance_name`: The name of the VM instance.
```

Comment:
How are you generating this list of metrics? I find it a bit redundant with `metricbeat/docs/fields.asciidoc`. If we feel the need of having the metrics descriptions in two places maybe we have to rethink how we generate these docs.
In any case they should be auto-generated from the `fields.yml` files, so we only have to maintain the descriptions in one place.

Reply:
I'm manually copying these fields in here so it can be displayed on individual metricset doc: https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-googlecloud-compute.html

Good point! I shouldn't have to copy paste these descriptions twice and maintain in two different places. Maybe in the doc, I should just display `x-pack/metricbeat/module/googlecloud/compute/_meta/fields.yml` instead?

Reply:
There is already a link to exported fields in the metricset docs.
![Captura de pantalla de 2020-05-28 20-37-49](https://user-images.githubusercontent.com/15763/83180047-2252cb00-a123-11ea-8669-248997b4e113.png)

But the one for this module seems quite empty: https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-googlecloud.html

We will have to review this. If we think that the fields should appear directly in the metricset docs page, we should generate them automatically there. If we want to continue using the separated page we will have to check why the fields of this module doesn't appear there.

Reply:
There is an old issue discussing about fields descriptions in docs: https://github.com/elastic/beats/issues/9288

Reply:
Thank you! I will create a separate PR to fix the `exported fields` section.